### ISSUE TITLE
Microsoft Office apps not appearing in search results

### ISSUE THREAD

**Issue by pgmccann** [mas-cli/mas#220]
**Describe the bug**
Can't find Microsoft Outlook, Microsoft Excel, Microsoft Word, Microsoft PowerPoint

**To Reproduce**
Run `mas search 'Microsoft Outlook'` (or equivalent for other apps)

**Expected behavior**
To see 'Microsoft Outlook' among the search results (the Office apps haveing recently been added to the MAS).

**Actual behavior**
A list of related apps, but not the application itself:

```
   582581625  MailTab for Outlook                                                                                          (2.3)
   973213640  MSG Viewer for Outlook                                                                                       (3.98.1)
   984397863  Inbox for Outlook                                                                                            (1.6)
   995017372  Mail+ for Outlook                                                                                            (1.9)
   592130193  MailTab Pro for Outlook                                                                                      (2.3)
  1073031650  Mail for Outlook                                                                                             (1.3)
   944792709  App for Outlook & Hotmail                                                                                    (1.0)
  1101267774  HTML Email Signature - Outlook                                                                               (1.4)
  1151461448  CalendarsPlus                                                                                                (1.6.2)
  1274576533  MSG Viewer for Outlook Pro                                                                                   (3.98.1)
   622372664  Tab for Outlook                                                                                              (1.0)
  1200374943  CalendarPro for Outlook                                                                                      (1.1)
   991390160  CompanionLink Sync for Outlook                                                                               (8.0.1023)
   958515704  App for OUTLOOK                                                                                              (1)
  1071154217  Menu Tab - for Microsoft Office  Quickly access Word, Excel, Powerpoint and Outlook from menu bar tab window (1.0)
  1177374768  Attachment Downloader for Outlook                                                                            (1.0)
  1067705033  Converter for Email to Outlook                                                                               (2.0.2)
```

**Desktop (please complete the following information):**
 - mas version: 1.6.2
 - macOS version: 10.14.3
 - How did you install mas? brew

**Comment by BarryMode** [comment]
I also just noticed this. I was coming here to see if there is a way to update the search results (although I assume they are live results in which case there is no way). I found another one that is also missing. Unsure if it is related to this but I'll go ahead and share. It's a Safari extension called PiPer.

`mas search piper` does not return the results, but `mas info 1421915518` works fine. Again, only sharing in case it helps with debugging the search.

**Comment by Zeta611** [comment]
These are the identifiers for Microsoft Office 365:
- Word: 462054704
- Powerpoint: 462062816
- Excel: 462058435
- Outlook: 985367838

`mas info` and `mas install` works fine with above identifiers.

**Comment by maxcbc** [comment]
These are also not appearing in `mas list` and `brew bundle dump`.
Might be worth raising as a separate issue, but when I've run into this also missing from the above commands' outputs are 
- Slack: 803453959 
- WhatsApp: 1147396723
- OneDrive: 823766827
Although these each do appear in Search.

**Comment by fharper** [comment]
I have the same issue with 1.6.3.

**Comment by phatblat** [comment]
This is certainly a bug. I think it might be related to the limit on search results as there is a lot of junk that shows up when you search for these popular Microsoft apps. OneNote seems to be the only one working.

```
↪ mas search microsoft
   715768417  Microsoft Remote Desktop 8                                                             (8.0.44)
   784801555  Microsoft OneNote                                                                      (16.31)
  1295203466  Microsoft Remote Desktop 10                                                            (10.3.5)
  1038079327  Docs² | for Microsoft Office                                                           (5.0)
...
```

## Workaround

For now use the app ID to install these apps directly and avoid the `search` and `lucky` commands. The app ID can be found by opening the App Store app, searching for the app, right-click and select "Copy Link". 

![Screen Shot 2019-11-26 at 11 20 51 AM](https://user-images.githubusercontent.com/28851/69661020-cfe30580-103e-11ea-8eb7-7e39df68b25d.png)

Then, extract the app ID number from `id#########` near the end of the URL

https://apps.apple.com/us/app/microsoft-word/id462054704?mt=12 -> 462054704

```
↪ mas info 985367838
Microsoft Outlook 16.31 [0.0]
By: Microsoft Corporation
Released: Nov 14, 2019
Minimum OS: 10.13
Size: 800.5 MB
From: https://apps.apple.com/us/app/microsoft-outlook/id985367838?mt=12&uo=4

↪ mas info 462054704
Microsoft Word 16.31 [0.0]
By: Microsoft Corporation
Released: Nov 14, 2019
Minimum OS: 10.13
Size: 1.06 GB
From: https://apps.apple.com/us/app/microsoft-word/id462054704?mt=12&uo=4

↪ mas info 462058435
Microsoft Excel 16.31 [0.0]
By: Microsoft Corporation
Released: Nov 14, 2019
Minimum OS: 10.13
Size: 816.4 MB
From: https://apps.apple.com/us/app/microsoft-excel/id462058435?mt=12&uo=4

↪ mas info 462062816
Microsoft PowerPoint 16.31 [0.0]
By: Microsoft Corporation
Released: Nov 14, 2019
Minimum OS: 10.13
Size: 732.6 MB
From: https://apps.apple.com/us/app/microsoft-powerpoint/id462062816?mt=12&uo=4

↪ mas info 784801555
Microsoft OneNote 16.31 [0.0]
By: Microsoft Corporation
Released: Nov 14, 2019
Minimum OS: 10.13
Size: 428.1 MB
From: https://apps.apple.com/us/app/microsoft-onenote/id784801555?mt=12&uo=4

↪ mas info 823766827
OneDrive 19.152.1013 [0.0]
By: Microsoft Corporation
Released: Oct 17, 2019
Minimum OS: 10.12
Size: 47.4 MB
From: https://apps.apple.com/us/app/onedrive/id823766827?mt=12&uo=4
```

**Comment by phatblat** [comment]
This is fixed now in mas 1.6.4

```
↪ mas search microsoft
   462054704  Microsoft Word                                                                         (16.37)
   784801555  Microsoft OneNote                                                                      (16.37)
   462058435  Microsoft Excel                                                                        (16.37)
   462062816  Microsoft PowerPoint                                                                   (16.37)
   985367838  Microsoft Outlook                                                                      (16.37)
  1274495053  Microsoft To Do                                                                        (2.17)
  1295203466  Microsoft Remote Desktop                                                               (10.3.10)
```
